plugins/inventory/icinga2.py

```diff
@@ -16,6 +16,8 @@
         - Get inventory hosts from the Icinga2 API.
         - "Uses a configuration file as an inventory source, it must end in
           C(.icinga2.yml) or C(.icinga2.yaml)."
+    extends_documentation_fragment:
+        - constructed
     options:
```

Comment:
This way the options will be clearly marked as added in 4.4.0:
```suggestion
    options:
      strict:
        version_added: 4.4.0
      compose:
        version_added: 4.4.0
      groups:
        version_added: 4.4.0
      keyed_groups:
        version_added: 4.4.0
```
(There are two more options, `use_extra_vars` and `leading_separator`, but these are only available if users use ansible-core 2.11+, but not if they use ansible-base 2.10 or Ansible 2.9. So it's better not to mention them here since the collection also supports these two versions.)